pipelines/rh-push-to-external-registry/README.md

```diff
@@ -0,0 +1,135 @@
+# Push to External Registry Pipeline
```

Comment:
This should be updated

Reply:
updated. but let me know what I am missing or not saying?

Reply:
It was identical to the push-to-external-registry one. This is a different pipeline. It should have its own name